This patch file is from Qualcomm Innovation Center, Inc. and is provided under the following license:
Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
SPDX-License-Identifier: BSD-3-Clause-Clear

From c55746308757777712b29110faa62dc5641aaff8 Mon Sep 17 00:00:00 2001
From: Tushar Darote <quic_tdarote@quicinc.com>
Date: Tue, 13 Feb 2024 09:36:59 +0000
Subject: [PATCH] Tensorflow-lite: XNNPACK and FP16 codebase github to CLO
 migration

For tflite v2.8, github to clo link update for XNNPACK and FP16

Signed-off-by: Tushar Darote <quic_tdarote@quicinc.com>
---
 tensorflow/lite/tools/cmake/modules/fp16_headers.cmake | 2 +-
 tensorflow/lite/tools/cmake/modules/xnnpack.cmake      | 2 +-
 tensorflow/workspace2.bzl                              | 2 +-
 third_party/FP16/workspace.bzl                         | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tensorflow/lite/tools/cmake/modules/fp16_headers.cmake b/tensorflow/lite/tools/cmake/modules/fp16_headers.cmake
index 0d039c01328..8ae97654269 100644
--- a/tensorflow/lite/tools/cmake/modules/fp16_headers.cmake
+++ b/tensorflow/lite/tools/cmake/modules/fp16_headers.cmake
@@ -22,7 +22,7 @@ include(FetchContent)
 
 OverridableFetchContent_Declare(
   fp16_headers
-  GIT_REPOSITORY https://github.com/Maratyszcza/FP16
+  GIT_REPOSITORY https://git.codelinaro.org/clo/le/external/github.com/maratyszcza/FP16
   # Sync with https://github.com/google/XNNPACK/blob/master/cmake/DownloadFP16.cmake
   GIT_TAG 0a92994d729ff76a58f692d3028ca1b64b145d91
   GIT_PROGRESS TRUE
diff --git a/tensorflow/lite/tools/cmake/modules/xnnpack.cmake b/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
index e6cd5fd7482..85394a70bc4 100644
--- a/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
+++ b/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
@@ -21,7 +21,7 @@ include(FetchContent)
 
 OverridableFetchContent_Declare(
   xnnpack
-  GIT_REPOSITORY https://github.com/google/XNNPACK
+  GIT_REPOSITORY https://git.codelinaro.org/clo/le/external/github.com/google/XNNPACK
   # Sync with tensorflow/workspace2.bzl
   GIT_TAG 113092317754c7dea47bfb3cb49c4f59c3c1fa10
   GIT_PROGRESS TRUE
diff --git a/tensorflow/workspace2.bzl b/tensorflow/workspace2.bzl
index 0129ff7f420..d4dd7ed336e 100644
--- a/tensorflow/workspace2.bzl
+++ b/tensorflow/workspace2.bzl
@@ -137,7 +137,7 @@ def _tf_repositories():
         name = "XNNPACK",
         sha256 = "8ce46328c9ac5f1b8359e5c4352a2bbafc39614eae7f647d32a6d43cb110ad01",
         strip_prefix = "XNNPACK-113092317754c7dea47bfb3cb49c4f59c3c1fa10",
-        urls = tf_mirror_urls("https://github.com/google/XNNPACK/archive/113092317754c7dea47bfb3cb49c4f59c3c1fa10.zip"),
+        urls = tf_mirror_urls("https://git.codelinaro.org/clo/le/external/github.com/google/XNNPACK/-/archive/113092317754c7dea47bfb3cb49c4f59c3c1fa10.zip"),
     )
     # LINT.ThenChange(//tensorflow/lite/tools/cmake/modules/xnnpack.cmake)
 
diff --git a/third_party/FP16/workspace.bzl b/third_party/FP16/workspace.bzl
index 4753d617959..5f45ff7538b 100644
--- a/third_party/FP16/workspace.bzl
+++ b/third_party/FP16/workspace.bzl
@@ -7,6 +7,6 @@ def repo():
         name = "FP16",
         strip_prefix = "FP16-4dfe081cf6bcd15db339cf2680b9281b8451eeb3",
         sha256 = "d973501a40c55126b31accc2d9f08d931ec3cc190c0430309a5e341d3c0ce32a",
-        urls = tf_mirror_urls("https://github.com/Maratyszcza/FP16/archive/4dfe081cf6bcd15db339cf2680b9281b8451eeb3.zip"),
+        urls = tf_mirror_urls("https://git.codelinaro.org/clo/le/external/github.com/maratyszcza/FP16/-/archive/4dfe081cf6bcd15db339cf2680b9281b8451eeb3.zip"),
         build_file = "//third_party/FP16:FP16.BUILD",
     )
-- 
2.35.7

